### ISSUE TITLE
\dx shows incorrect version number

### ISSUE THREAD

**Issue by robins** [dimitri/prefix#23]
Prefix still shows version as v1.2.0:

```
postgres=# \dx
                    List of installed extensions
  Name   | Version |   Schema   |            Description
---------+---------+------------+------------------------------------
 plpgsql | 1.0     | pg_catalog | PL/pgSQL procedural language
 prefix  | 1.2.0   | public     | Prefix Range module for PostgreSQL
(2 rows)

```

When I check the control file, it still shows v1.2.0. 

https://github.com/dimitri/prefix/blob/master/prefix.control

```
# prefix extension
comment = 'Prefix Range module for PostgreSQL'
default_version = '1.2.0'
module_pathname = '$libdir/prefix'
relocatable = true
```

It'd be great if \dx shows the correct version i.e. v.1.2.9.

**Comment by df7cb** [comment]
Hi,

the extension version in PostgreSQL is actually tracking the version number of the SQL interface of extension, what functions it provides, and function signatures. C-only code changes aren't reflected, the prefix.so library on disk can be upgraded without the need for the user typing `ALTER EXTENSION prefix UPDATE;`. This version number is more like the SONAME of C libraries than the actual project version number.

The proper fix here would be to use `1.2` as version number in SQL, we shall fix that the next time there's an actual SQL change.

Christoph

**Comment by ardentperf** [comment]
> The proper fix here would be to use `1.2` as version number in SQL, we shall fix that the next time there's an actual SQL change.

Seems reasonable - should we leave this issue open as a placeholder until we can tag it to the next SQL change?  Or is it tracked somewhere else?
